vpdq/cpp/bin/vpdq-hash-video.cpp

```diff
@@ -0,0 +1,196 @@
+#include <vector>
+#include <vpdq/cpp/hashing/filehasher.h>
+#include <vpdq/cpp/hashing/vpdqHashType.h>
+#include <vpdq/cpp/io/vpdqio.h>
+
+using namespace std;
+
+static void usage(char* argv0, int rc) {
+  FILE* fp = (rc == 0) ? stdout : stderr;
+  fprintf(fp, "Usage: %s [options]\n", argv0);
+  fprintf(fp, "Required:\n");
+  fprintf(fp, "-f|--ffmpeg-path ...\n");
+  fprintf(fp, "-i|--input-video-file-name ...\n");
+  fprintf(fp, "-o|--output-hash-file-name ...\n");
+  fprintf(fp, "-r|--seconds-per-hash ...:Must be a positive integer\n");
+  fprintf(fp, "Options:\n");
+  fprintf(fp, "-v|--verbose: Show all hash matching information\n");
+  fprintf(
+      fp,
+      "-d|--output-directory ...: instead of specifiying "
+      "output-file name, just give a directory and the output file name will "
```

Comment:
More details on how it will be computed? Something like "avideofile.mp4 will result in <output_directory>/avideofile.mp4.vpdq.txt."